content/en/docs/concepts/storage/volume-attributes-classes.md

```diff
@@ -8,7 +8,7 @@ weight: 40
 ---
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.29" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.31" state="beta" >}}
```

Comment:
Improvement for 'feature-state' shortcode usage here, consider using the new feature_gate_name option, making the feature state more data-driven.

```suggestion
{{< feature-state feature_gate_name="VolumeAttributesClass" >}}
```